src/Console/ModelsCommand.php

```diff
@@ -805,17 +812,34 @@ protected function createPhpDocs($class)
         }
 
         $serializer = new DocBlockSerializer();
-        $serializer->getDocComment($phpdoc);
```

Comment:
lol, thx

Reply:
Yes, initially thought it was some kind of hack, but it is just a forgotten line.